docs/development/maintenance.md

```diff
@@ -13,1292 +15,1464 @@ modules of ILIAS.
-	* 1st Maintainer: [akill](https://docu.ilias.de/goto_docu_usr_149.html)
-	* 2nd Maintainer: N.A.
-	* Testcases: [atoedt](https://docu.ilias.de/goto_docu_usr_3139.html)
+    * Authority to Sign off Code Changes: [akill](https://docu.ilias.de/goto_docu_usr_149.html)
+    * Authority to Sign off Conceptual Changes: [akill](https://docu.ilias.de/goto_docu_usr_149.html)
+    * Authority to Sign off Testcase Changes: [atoedt](https://docu.ilias.de/goto_docu_usr_3139.html)
+    * Authority to (De-)Assign Authoritiees: [akill](https://docu.ilias.de/goto_docu_usr_149.html)
 	* Tester: [miriamwegener](https://docu.ilias.de/goto_docu_usr_23051.html)
+    * Unit-specific Guidelines, Rules, and Regulations: [LINK MISSING]('')
 
 [//]: # (END Exercise)
 
 [//]: # (BEGIN Export)
 
 * **Export**
-	* Maintenance Model: Classic
-	* 1st Maintainer: [smeyer](https://docu.ilias.de/goto_docu_usr_191.html)
-	* 2nd Maintainer: N.A.
-	* Testcases: [Fabian](https://docu.ilias.de/goto_docu_usr_27631.html)
+    * Authority to Sign off Code Changes: [smeyer](https://docu.ilias.de/goto_docu_usr_191.html)
+    * Authority to Sign off Conceptual Changes: [smeyer](https://docu.ilias.de/goto_docu_usr_191.html)
+    * Authority to Sign off Testcase Changes: [Fabian](https://docu.ilias.de/goto_docu_usr_27631.html)
+    * Authority to (De-)Assign Authoritiees: [smeyer](https://docu.ilias.de/goto_docu_usr_191.html)
 	* Tester: [Fabian](https://docu.ilias.de/goto_docu_usr_27631.html)
+    * Unit-specific Guidelines, Rules, and Regulations: [LINK MISSING]('')
 
 [//]: # (END Export)
 
 [//]: # (BEGIN Favourites)
 
 * **Favourites**
-	* Maintenance Model: Classic
-	* 1st Maintainer: [akill](https://docu.ilias.de/goto_docu_usr_149.html)
-	* 2nd Maintainer: N.A.
-	* Testcases: [AUTHOR MISSING](https://docu.ilias.de/goto_docu_pg_64423_4793.html)
+    * Authority to Sign off Code Changes: [akill](https://docu.ilias.de/goto_docu_usr_149.html)
+    * Authority to Sign off Conceptual Changes: [akill](https://docu.ilias.de/goto_docu_usr_149.html)
+    * Authority to Sign off Testcase Changes: [akill](https://docu.ilias.de/goto_docu_usr_149.html)
+    * Authority to (De-)Assign Authoritiees: [akill](https://docu.ilias.de/goto_docu_usr_149.html)
 	* Tester: [TESTER MISSING](https://docu.ilias.de/goto_docu_pg_64423_4793.html)
+    * Unit-specific Guidelines, Rules, and Regulations: [LINK MISSING]('')
 
 [//]: # (END Favourites)
 
 [//]: # (BEGIN File)
 
 * **File**
-	* Maintenance Model: Classic
-	* 1st Maintainer: [fschmid](https://docu.ilias.de/goto_docu_usr_21087.html)
-	* 2nd Maintainer: N.A.
-	* Testcases: [scarlino](https://docu.ilias.de/goto_docu_usr_56074.html)
+    * Authority to Sign off Code Changes: [fschmid](https://docu.ilias.de/goto_docu_usr_21087.html)
+    * Authority to Sign off Conceptual Changes: [fschmid](https://docu.ilias.de/goto_docu_usr_21087.html)
+    * Authority to Sign off Testcase Changes: [scarlino](https://docu.ilias.de/goto_docu_usr_56074.html)
```

Comment:
The Profile behind [scarlino](https://docu.ilias.de/goto_docu_usr_56074.html) is not active, maybe change to fschmid?